write-rustdoc-hide-lines/README.md

```diff
@@ -0,0 +1,15 @@
+# Write `rustdoc` `hide_lines` annotations
+
+This utility iterates over the Markdown files on the given folder.
+Searches `rust` code blocks and ensures that the [`hide_lines` Zola annotation](https://www.getzola.org/documentation/content/syntax-highlighting/#annotations) match the code block "hidden" lines (the ones starting with `#`).
```

Comment:
I would prefer a reference-style link instead, to not break the flow of the sentence.

Reply:
You mean something like this, no? 👉  https://github.com/bevyengine/bevy-website/pull/405/commits/b1ded3b9f46d6fcaff48763422ac815fa3d4ab71

Reply:
Yes. If you want, you can also get rid of the index and use the anchor text as reference.

Reply:
Done!